### PR TITLE
Fix: Infantry Muzzle Flash Stuck During Transition Animations, part 2

### LINKED ISSUE #2163: First Missile Defender in a session may display muzzle flash when paradropped
https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/0d8958dc-ddfc-4557-b80b-e78695ddc2a3

Seems similar to https://github.com/TheSuperHackers/GeneralsGamePatch/pull/1078

Fix is probably similar to https://github.com/TheSuperHackers/GeneralsGamePatch/pull/938

What other units should I go through? All of them, just to be sure?

### PR DESCRIPTION
**Pull request by commy2**
- continue https://github.com/TheSuperHackers/GeneralsGamePatch/pull/938
- fix https://github.com/TheSuperHackers/GeneralsGamePatch/issues/2163

### CODE REVIEW

**Comment by commy2** [comment]
- [x] AirF_AmericaInfantryColonelBurton
- [x] AirF_AmericaInfantryMissileDefender
- [x] AirF_AmericaInfantryPathfinder
- [x] AirF_AmericaInfantryRanger
- [x] AmericaInfantryCIAOfficer
- [x] AmericaInfantryColonelBurton
- [x] AmericaInfantryMissileDefender
- [x] AmericaInfantryOfficer
- [x] AmericaInfantryPathfinder
- [x] AmericaInfantryRanger
- [x] AmericaInfantrySecretService
- [x] Boss_InfantryColonelBurton
- [x] Boss_InfantryJarmenKell
- [x] Boss_InfantryPathfinder
- [x] Boss_InfantryRanger
- [x] Boss_InfantryTankHunter
- [ ] Chem_GLAInfantryAngryMobPistol01
- [x] Chem_GLAInfantryJarmenKell
- [x] Chem_GLAInfantryRebel
- [x] Chem_GLAInfantryStingerSoldier
- [x] Chem_GLAInfantryTunnelDefender
- [x] ChinaInfantryAgent
- [x] ChinaInfantryOfficer
- [x] ChinaInfantryParadeRedGuard
- [x] ChinaInfantryRedguard
- [x] ChinaInfantryTankHunter
- [ ] Demo_GLAInfantryAngryMobPistol01
- [x] Demo_GLAInfantryJarmenKell
- [x] Demo_GLAInfantryRebel
- [x] Demo_GLAInfantryStingerSoldier
- [x] Demo_GLAInfantryTunnelDefender
- [x] GC_Chem_GLAInfantryJarmenKell
- [x] GC_Chem_GLAInfantryRebel
- [x] GC_Chem_GLAInfantryStingerSoldier
- [x] GC_Chem_GLAInfantryTunnelDefender
- [x] GC_Slth_GLAInfantryJarmenKell
- [x] GC_Slth_GLAInfantryRebel
- [x] GC_Slth_GLAInfantrySniper
- [x] GC_Slth_GLAInfantryStingerSoldier
- [x] GC_Slth_GLAInfantryTunnelDefender
- [x] GC_Slth_GLAInfantryWorker
- [x] GLAInfantryAngryMobPistol01
- [x] GLAInfantryJarmenKell
- [x] GLAInfantryRebel
- [x] GLAInfantryStingerSoldier
- [x] GLAInfantryTunnelDefender
- [x] GLAInfantryWorker
- [x] Infa_ChinaInfantryMiniGunner
- [x] Infa_ChinaInfantryTankHunter
- [x] Lazr_AmericaInfantryColonelBurton
- [x] Lazr_AmericaInfantryMissileDefender
- [x] Lazr_AmericaInfantryPathfinder
- [x] Lazr_AmericaInfantryRanger
- [x] MilitiaMan
- [x] Nuke_ChinaInfantryRedguard
- [x] Nuke_ChinaInfantryTankHunter
- [x] Partisan01
- [x] Partisan02
- [x] Partisan03
- [ ] Slth_GLAInfantryAngryMobPistol01
- [x] Slth_GLAInfantryJarmenKell
- [x] Slth_GLAInfantryRebel
- [x] Slth_GLAInfantryStingerSoldier
- [x] Slth_GLAInfantryTunnelDefender
- [x] SupW_AmericaInfantryColonelBurton
- [x] SupW_AmericaInfantryMissileDefender
- [x] SupW_AmericaInfantryPathfinder
- [x] SupW_AmericaInfantryPilot
- [x] SupW_AmericaInfantryRanger
- [x] Tank_ChinaInfantryRedguard
- [x] Tank_ChinaInfantryTankHunter
- [x] UNSoldier

**Comment by commy2** [comment]
### before

shot_20230822_214429_3

**Comment by commy2** [comment]
Burton has this fix natively in 1.04.

**Comment by commy2** [comment]
Done, ready.

File: Patch104pZH/Design/Changes/v1.0/2194_dead_plane_helicopter_bug.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ labels:
 links:
   - https://github.com/TheSuperHackers/GeneralsGamePatch/pull/1018
   - https://github.com/TheSuperHackers/GeneralsGamePatch/pull/2194
+  - https://github.com/TheSuperHackers/GeneralsGamePatch/pull/2268
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/2194_dead_plane_helicopter_bug.yaml]: Is this correct?

Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/2194_dead_plane_helicopter_bug.yaml]: Yes (no)

**Review by xezon — APPROVED** [review]

**Pull request merged**